WordPressShared/Core/Analytics/WPAnalytics.h

```diff
@@ -110,6 +110,12 @@ typedef NS_ENUM(NSUInteger, WPAnalyticsStat) {
     WPAnalyticsStatGravatarCropped,
     WPAnalyticsStatGravatarTapped,
     WPAnalyticsStatGravatarUploaded,
+    WPAnalyticsStatGutenbergWarningConfirmDialogShown,
+    WPAnalyticsStatGutenbergWarningConfirmDialogShownYesTapped,
+    WPAnalyticsStatGutenbergWarningConfirmDialogShownCancelTapped,
+    WPAnalyticsStatGutenbergWarningConfirmDialogShownDontShowAgainChecked,
+    WPAnalyticsStatGutenbergWarningConfirmDialogShownDontShowAgainUnchecked,
+    WPAnalyticsStatGutenbergWarningConfirmDialogShownLearnMoreTapped,
```

Comment:
Maybe remove `Shown` from the names of all but `WPAnalyticsStatGutenbergWarningConfirmDialogShown` since its an action.